### DISCUSSION TITLE
Pinia awesome

### DISCUSSION THREAD

**Discussion by reslear** [vuejs/pinia#887] [Ideas]
Create `Pinia awesome` with community plugin collection in doc page 

like schema: 

### Storage
- [pinia-plugin-persistedstate-2](https://github.com/iendeavor/pinia-plugin-persistedstate-2) - persisted state vue 2-3 and nuxt 2

---
more - example - https://github.com/vitejs/awesome-vite

**Comment by posva** [comment]
Sorry for the late reply. I think a new repo is too much work. A page for community plugins in docs would be nice.

Also, I think there are too many different persisted state plugins 🤪
